### PR TITLE
docs: update `useParams` hook return statement

### PR DESCRIPTION
**Pull request by Kuboczoch**
Hi 👋🏻 

## The Change

I noticed that in one example of the [`useParams` hook](https://nextjs.org/docs/app/api-reference/functions/use-params), there is an old-fashioned return statement that returns a react fragment instead of null. This step was part of the workaround from the typescript environment from 2016, which is now obsolete.

Recently, I researched why some people do it today and whether we should continue doing it in a react environment.
I created a document discussing it:
https://github.com/Kuboczoch/web-dev-best-practices/blob/master/react/conditional-rendering_return-react-fragment.md

tl;dr
> While returning a `React.Fragment` instead of `null` in a React component was a workaround for TypeScript limitations in 2016, this is no longer necessary. Returning `null` is more efficient, makes the code more readable, is officially supported, and aligns with current best practices.

## Why?

It helps to provide the best practice to other developers who are just starting with web development and are mindlessly copy-pasting the code.  In this case, people do not need to research workarounds for TypeScript from six years ago.

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: aa5baea64c3a658ac725fe5bc54f7bcbacd27ef4

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/02-app/02-api-reference/04-functions/use-params.mdx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default function ExampleClientComponent() {
   // `params` -> { tag: 'shoes', item: 'nike-air-max-97' }
   console.log(params)
 
-  return <></>
+  return null
```
Comment (devjiwonchoi) [inline docs/02-app/02-api-reference/04-functions/use-params.mdx:21]: Since it is an example component, it seems like the fragment was used to abstract the rest code, not just returning null.

How about something like this?

```suggestion
  return <>{/* ...rest */}</>
```

Comment (Kuboczoch) [reply docs/02-app/02-api-reference/04-functions/use-params.mdx:21]: My example was more about what happens when you don't return anything. I wanted to keep the functionality the same as for the previous docs.
In my real-life production code, we don't expect anything in the return statement anyway for this component, and the `<></>` react fragment is confusing for new developers.

But your change is good too!
Can we do it in the next pr?

Let me know what you think!

Comment (devjiwonchoi) [reply docs/02-app/02-api-reference/04-functions/use-params.mdx:21]:
```suggestion
  return '...'
```

Comment (devjiwonchoi) [reply docs/02-app/02-api-reference/04-functions/use-params.mdx:21]: Let's go with this! Will further use`'...'` throughout the examples instead of `null` or `<></>` 😄

**Review by devjiwonchoi — CHANGES_REQUESTED** [review]
Let's go with `return '...'`!

https://github.com/vercel/next.js/pull/67384/files#r1664243618

**Comment by delbaoliveira** [comment]
Thank you for sharing @Kuboczoch, I wasn't aware that `<></>` was used as TS workaround for returning null. Like @devjiwonchoi mentioned, we use it to indicate that more that can be returned from the component, not to return `null`.  We'll keep that in mind in the future.

File: docs/02-app/02-api-reference/04-functions/use-params.mdx (hunk 1 above)
Comment (delbaoliveira) [inline docs/02-app/02-api-reference/04-functions/use-params.mdx:21]:
```suggestion
  return '...'
```

File: docs/02-app/02-api-reference/04-functions/use-params.mdx [hunk 2]
```diff
@@ -35,7 +35,7 @@ export default function ExampleClientComponent() {
...
```
Comment (delbaoliveira) [inline docs/02-app/02-api-reference/04-functions/use-params.mdx:38]:
```suggestion
  return '...'
```

**Comment by Kuboczoch** [comment]
> Thank you for sharing @Kuboczoch, I wasn't aware that `<></>` was used as TS workaround for returning null. Like @devjiwonchoi mentioned, we use it to indicate that more that can be returned from the component, not to return `null`.  We'll keep that in mind in the future. 

Thanks, regarding the pr I will update it when I have a free time and request a review once more 👍

**Review by devjiwonchoi — APPROVED** [review]
Thank you for the PR!
